listenbrainz/webserver/views/api.py

```diff
@@ -76,6 +76,9 @@ def submit_listen():
     if mb_engine and current_app.config["REJECT_LISTENS_WITHOUT_USER_EMAIL"] and not user["email"]:
         raise APIUnauthorized(REJECT_LISTENS_WITHOUT_EMAIL_ERROR)
 
+    if user['is_paused']:
```

Comment:
I think instead of user_id, the error should say the user account. also same for the error message in the other two api files.